assets/js/contact.js

```diff
@@ -1,22 +1,38 @@
-function bindCaptchaValidation() {
+function verifyCanSubmitEntry() {
   const debug = Array.prototype.slice.apply(document.getElementsByName('debug'))[0];
   if (debug && +debug.value) {
     return;
   }
 
   const form = document.getElementById('contact-us-form');
   const error = document.getElementById('captcha-error-message');
-
+  const piiError = document.getElementById('pii-warning');
+  const piiErrorText = document.getElementById('pii-warning-message');
+  const descriptionInput = document.getElementById('description');
+  let counter = 0;
   form.addEventListener('submit', (event) => {
     const captcha = document.getElementById('g-recaptcha-response');
     if (!captcha || !captcha.value) {
       event.preventDefault();
       error.textContent = error.dataset.error;
       error.classList.remove('display-none');
+    } else if (descriptionInput.value.match(/\d{4,}/) && counter < 1) {
```

Comment:
I would consider extracting this into a function
1. lets us test it
2. is a clear place to add more complex checks in the future if we want to add those
```suggestion
    } else if (containsPii(descriptionInput.value) && counter < 1) {
```